smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/unique-items-floats.errors

```diff
@@ -0,0 +1,6 @@
+[WARNING] smithy.example#Floats: This shape marked with @uniqueItems contains a float in its closure (`smithy.api#Float`). Equality comparisons for floating point numbers or document types that can contain floating point numbers can be problematic. | UniqueItems
+[WARNING] smithy.example#Doubles: This shape marked with @uniqueItems contains a double in its closure (`smithy.api#Double`). Equality comparisons for floating point numbers or document types that can contain floating point numbers can be problematic. | UniqueItems
+[WARNING] smithy.example#Documents: This shape marked with @uniqueItems contains a document in its closure (`smithy.api#Document`). Equality comparisons for floating point numbers or document types that can contain floating point numbers can be problematic. | UniqueItems
+[WARNING] smithy.example#Structures: This shape marked with @uniqueItems contains a document in its closure (`smithy.api#Document`). Equality comparisons for floating point numbers or document types that can contain floating point numbers can be problematic. | UniqueItems
```

Comment:
Should this be pointing to the member or is it a duplicate?

Reply:
I made it so warnings for the same shape are grouped now.